<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Smitha M Shankar</title>
<script type="text/javascript">
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link href="sitestyle.css" rel="stylesheet" type="text/css" />
</head>

<body onload="MM_preloadImages('nav-tap-rollover.jpg','Courses-rollover.jpg','Home-Rollover.jpg','Research-Rollover.jpg','Projects-Rollover.jpg')">
<div><img src="iub_white.gif" width="171" height="44" hspace="200" />
</div>
<div><img src="background.gif" width="15%" height="65" /><img src="soic.gif" width="40%" height="65" /><img src="background.gif" alt="" width="45%" height="65" /></div>
<div id="background"><img src="nav=top.jpg" width="14.5%" height="36" /><a href="home.html" onmouseover="MM_swapImage('home','','Home-Rollover.jpg',1)" onmouseout="MM_swapImgRestore()"><img src="Home.jpg" name="home" width="73" height="36" border="0" id="home" /></a><img src="Research-Rollover.jpg" width="106" height="36" /><a href="projects.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('projects','','Projects-Rollover.jpg',1)"><img src="Projects.jpg" name="projects" width="91" height="36" border="0" id="projects" /></a><a href="courses.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Courses','','Courses-rollover.jpg',1)"><img src="Courses.jpg" name="Courses" width="90" height="36" border="0" id="Courses" /></a><img src="nav=top.jpg" alt="" width="58.9%" height="36" /></div>
<div><img src="nav-top-shadow.jpg" width="100%"/></div>
<div id="background">
	<div id="swirls-1">
    	<div id="swirls-2">
        	<div id="main-content" align="center">
            	<br />
                <h3 align="left">•	Expression Analysis of Microarray Data
				</h3>				
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Worked on Secondary Analysis and Statistical Analysis of raw Microarray Data to obtain significant Differential Expression, primarily for Nimblegen Arrays. This process involves Image analysis, Preprocessing of data, Data Normalization, Linear Regression Analysis and Statistical Significance Analysis. Familiarity in both expression and Tiling Arrays. Also presented a tutorial on Microarray Data Analysis for Nimblegen Array Data.
				</h5></br>
                <h3 align="left">•	Pipeline Analysis of Next-Generation Sequencing data</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				including Mapping, Assembly, SNP detection and Genome Annotation for both Illumina and 454 datasets. Used tools like Neubler, Tophat, Cufflinks, Bowtie, BWA.
				</h5></br>
                <h3 align="left">•	Investigation the effect of environmental factors on gene expression on Daphnia pulex</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Performed microarray analysis on 110 datasets and compared expression levels from a polluted and unpolluted lake and identified differentially expressed genes in response to Cadmium, Nickel, Copper, Arsenic, Acetaminophen and a Herbicide and observed significant biases. Currently performing Principal Component Analysis on the mixture of treatments.
				</h5></br>
                <h3 align="left">•	Analysis of copy number variants in breast cancer and ovarian cancer using array CGH</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Calculated and compared log2 changes in genes for healthy and cancer samples to identify potential gene gains and loses. This dataset involves breast and ovarian cancer data from 3 cell lines for each sample- MCF7, MCF7F, MCF7T, NOSE,OCIC, BULK.
				</h5></br>		
                <h3 align="left">•	Discovery of enriched functional modules from microarray data for Daphnia pulex.</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Used GSEA and GENOMICA to identify enriched gene-sets and modules for a 110 samples form a polluted and unpolluted lake under 6 different chemical treatments as a part of a class project.
				</h5></br>
                <h3 align="left">•	Construction of a phylogenetic tree based on 16s rRNA gene copy number</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Aligned 16srrna sequences from 574 microbial species and built a novel copy number based phylogenetic tree, this tree now being used to classify new species obtained from metagenomics samples by mapping the copy number location on the tree.
				</h5></br>
                <h3 align="left">•	Insilico Design of a Pancreatic Lipase Inhibitor-Indgene Labs, Bangalore, India-Intern</h3>
                <h4 align="left">Adivsor Name and Year of Research</h4>
				<h5 align="left">
				Researched and formulated an anti-obesity drug insilico. This involved formulating 50 potential drug, and docked these molecules using AUTODOCK3, and obtained 5 molecules which could be used for lipase inhibition. Presented a paper based on this research at an 
International Biotechnology Conference “Molecular Mechanisms and Systems Biology” in Bangalore, India
				</h5></br>
		  </div>
        </div>
    </div>
</div>
</body>
</html>
