<html>
<head>
<link rel="stylesheet" href="mycss.css">
<hr/>
<h2 align="center">Research</h2>
<hr/>
</head>
<body>
<u><h3>Spatial Datatypes and Moving Objects Databases</h3></u>
<ul>
<li><b>PAR-NJ [2010]</b>
	<br/>
	Language: Java
	<br/><br/>
	PAR-NJ is a one of a kind program that parallels the process of protein sequence alignment and phylogenetic tree construction. 
	The program accepts a file of protein sequences(tested for upto 5000 protein sequences) and distributes the process of computation of global alignment between
	the cores of a processor. It even parallels the process of phylogenetic tree construction. Needleman-Wunsch algorithm is used for global sequence alignment, and
	Neighbor-Joining algorithm is used for guide tree construction. The project also involved comparing the performance of the a prominent phylogenetic 
	tree construction program called QuickTree. More details about this project can be found <a href="http://www.par-nj.webs.com/">here</a>.
	<br/><br/>
<li><b>Protein Phylogenetic Tree Construction [2010]</b>
	<br/>
	Language: C++
	<br/><br/>
	I developed a program that takes as input, a file of protein sequences and use the Unweighted Pair Group Method with Arithematic mean(UPGMA) algorithm 
	to generate a phylogenetic tree in the Newick format. The code structure would be later used in the PAR-NJ project described above.
	<br/><br/>
<li><b>DNA Sequence Alignment [2010]</b>
	<br/>
	Language: Java
	<br/><br/>
	This project was mostly built to be able to scale the process of DNA sequence alignment over very large datasets(upto 40,000 DNA sequences). The primary idea is 
	to align the seqeunces using the Needleman-Wunsch global alignment algorithm. This project would later be used in the development of PAR-NJ described above.
	<br/><br/>
</ul>
<hr/>
</body>
</html>